.gitignore

```diff
@@ -28,3 +28,7 @@ Session.vim
 *~
 tags
 .pem
+
+
+# Post-test cruft
+Gemfile.lock
```

Comment:
I think we actually still want this file tracked. It's the result of using bundler, and it will allow us to make sure we're using exactly what Github uses when they build our site. It's a lot like upper-constraints.txt in pip. Basically you typically will want to use bundle exec to build or serve while testing changes locally:

```$ bundle exec jekyll ...```

If you don't have a Gemfile.lock yet, and you ```bundle install``` then it will use the highest versions of everything from Gemfile.